### DISCUSSION TITLE
【linux】按照README编译产出了vender目录，想问下如何使用

### DISCUSSION THREAD

**Discussion by MarvinBaidu** [Tencent/libpag#2417] [Q&A] [answered]
麻烦问下如何：
1、读取.pag模板
2、替换占位图
3、替换文字
4、渲染导出视频

**Comment by kevingpqi123** [comment, marked as answer]
建议看下源码中 linux 文件夹，下面有 demo，上面有编辑文本和替换占位图，可以获取每一帧的渲染数据，渲染导出视频不是 pag 的范围，需要自己去开发
